### ISSUE TITLE
Newcomers_Guide.rst: Add example of assigning an issue

### ISSUE THREAD

**Issue by yash-nisar** [coala/coala#3845]
Many times, people make a mistake and do not use the full issue url while assigning an issue via cobot. Perhaps adding an example could help.
difficulty/newcomer
area/documentation

**Comment by supergr35** [comment]
Do you think we should just add a general cobot documentation which includes this?

**Comment by yash-nisar** [comment]
Yeah, sounds good, I think there should be an example IMO because most of the people just glance through the docs, without reading the specifics. The example may catch their attention.

**Comment by Anirudh5** [comment]
@yash-nisar I want to take up this issue.

**Comment by yash-nisar** [comment]
@Anirudh5 You can assign an issue to yourself by `cobot assign full-issue-url` on our gitter channel.

**Comment by whosethink** [comment]
I have excute "coala --files=cobot_help.rst --bears=SpaceConsistencyBear -I" , but still have the problem of  trailing whitespaces, can anyone tell how to handle this problem?
![2017-03-07 00-47-27](https://cloud.githubusercontent.com/assets/20852522/23619868/060a500c-02d0-11e7-8d2b-e8739db793a1.png)

**Comment by vedanshbhartia** [comment]
@whosethink Are you still working on this issue?

**Comment by jayvdb** [comment]
unassigned due to inactivity

**Comment by Makman2** [comment]
What is this issue blocked by? @jayvdb

**Comment by jayvdb** [comment]
@Makman2 , this is about how to use corobo to assign an issue.  That can not be done atm.

**Comment by jayvdb** [comment]
leaving it assigned to @nvzard who can unblock and unassign once his work has made this possible again.
